Dockerfile

```diff
@@ -2,14 +2,18 @@ FROM apache/superset:5.0.0rc2@sha256:00c4ddf3b8a6a9fc95f0dffb09e8f29129732ca5fee
 
 USER root
 
-COPY superset_config.py /app/pythonpath/
-RUN chown -R superset:superset /app/pythonpath
-RUN mkdir -p /app/owslib && chown -R superset:superset /app/owslib
+RUN apt-get update && apt-get install -y git && rm -rf /var/lib/apt/lists/*
```

Comment:
Why do you install git here? I guess i just don't see where you are using it.

Reply:
I needed it to install OWSLib via the git repo. This will no longer be necessary once a new official version of the library is available.